iotdb/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -359,14 +366,43 @@ public TSFetchMetadataResp fetchMetadata(TSFetchMetadataReq req) throws TExcepti
     return resp;
   }
 
+  protected Set<String> getAllStorageGroups() throws PathErrorException, InterruptedException {
+    return MManager.getInstance().getAllStorageGroup();
+  }
+
+  protected List<List<String>> getTimeSeriesForPath(String path)
+      throws PathErrorException, InterruptedException, ProcessorException {
+    return MManager.getInstance().getShowTimeseriesPath(path);
+  }
+
+  protected String getMetadataInString()
+      throws InterruptedException, PathErrorException, ProcessorException {
+    return MManager.getInstance().getMetadataInString();
+  }
+
+  protected Metadata getMetadata()
+      throws PathErrorException, InterruptedException, ProcessorException {
+    return MManager.getInstance().getMetadata();
+  }
+
+  protected TSDataType getSeriesType(String path)
+      throws PathErrorException, InterruptedException, ProcessorException {
+    return MManager.getInstance().getSeriesType(path);
+  }
+
+  protected List<String> getPaths(String path)
+      throws PathErrorException, InterruptedException, ProcessorException {
+    return MManager.getInstance().getPaths(path);
+  }
+
   /**
-   * Judge whether the statement is ADMIN COMMAND and if true, executeWithGlobalTimeFilter it.
+   * Judge whether the statement is ADMIN COMMAND and if true, execute it.
    *
    * @param statement command
    * @return true if the statement is ADMIN COMMAND
    * @throws IOException exception
    */
-  private boolean execAdminCommand(String statement) throws IOException {
+  public boolean execAdminCommand(String statement) throws IOException {
```

Comment:
why change to `public`?

Reply:
I have no idea why I change so many private to public of method. Thx. I have changed it back to private.